WooCommerce/Classes/Extensions/UITableViewCell+Helpers.swift

```diff
@@ -12,10 +12,37 @@ extension UITableViewCell {
         return classNameWithoutNamespaces
     }
 
-    /// Applies the default background color
-    ///
-    func applyDefaultBackgroundStyle() {
-        backgroundColor = .listForeground(modal: false)
+    private func defaultBackgroundConfiguration(style: UITableView.Style = .grouped) -> UIBackgroundConfiguration {
```

Comment:
Added this helper function for creating the configuration object to avoid duplicate code

Reply:
using .grouped as a default parameter but leaving option to sending a different one to calles